core/lib/makeswift/client.ts

```diff
@@ -1,11 +1,25 @@
 import { Makeswift } from '@makeswift/runtime/next';
+import { getSiteVersion } from '@makeswift/runtime/next/server';
 import { strict } from 'assert';
+import { getLocale } from 'next-intl/server';
 
+import { defaultLocale } from '~/i18n/routing';
 import { runtime } from '~/lib/makeswift/runtime';
 
+import '~/lib/makeswift/components';
+
 strict(process.env.MAKESWIFT_SITE_API_KEY, 'MAKESWIFT_SITE_API_KEY is required');
 
 export const client = new Makeswift(process.env.MAKESWIFT_SITE_API_KEY, {
   runtime,
   apiOrigin: process.env.MAKESWIFT_API_ORIGIN,
 });
+
+export const getComponentSnapshot = async (snapshotId: string) => {
```

Comment:
Encapsulate `getComponentSnapshot` wiring logic so that we don't spread this cruft everywhere.